src/open_inwoner/accounts/gateways.py

```diff
@@ -69,7 +69,7 @@ def send(self, to, token, **kwargs):
                 logger.critical(
                     ("Could not send SMS to {to}:\n{error}").format(to=to, error=error)
                 )
-            raise GatewayError()
+            raise GatewayError() from e
```

Comment:
```suggestion
            raise GatewayError() from exc
```
for consistency with the rest of our codeboase (requires change above of course)

Reply:
Check.